torch/csrc/autograd/functions/accumulate_grad.cpp

```diff
@@ -59,5 +60,45 @@ auto AccumulateGrad::apply(variable_list&& grads) -> variable_list {
   return variable_list();
 }
 
+#ifdef TORCH_COMPILED_AUTOGRAD
+void AccumulateGrad::compiled_args(CompiledNodeArgs& args) {
+  if (args.cond(
+          variable.defined() && variable.requires_grad() &&
+          GradMode::is_enabled())) {
```

Comment:
I don't think you want to check gradmode here? This would only be on if you're doing double backward?

Reply:
I'll remove this.